### ISSUE TITLE
Add support for Profiles

### ISSUE THREAD

**Issue by markched** [ClusterM/hakchi2#1050]
It would be cool to have hakchi profiles. for example if my brother and I both have classics and he wants to change or I want to change, rather than ticking and unticking or deleting and adding games you could just be like settings --> profile --> John Smith and BOOM your set up is there with all your games.

**Comment by Drakonas** [comment]
Just an FYI you can have multiple Hakchi portable installs for the time being to do what you want. Just make sure to copy the dump folder over to have the original kernel dump.

**Comment by markched** [comment]
I tried that and when I made changes in one it made the change in the other. Very weird. Maybe I need to rename the file? 

Is this something the community would like to have?

**Comment by ReyVGM** [comment]
You need to have two separate hakchi folders. But you cannot use the web installer, you have to download the portable version which does not need installing.

**Comment by markched** [comment]
makes sense! Thank you guys, and thank you @Drakonas for tagging this as a feature reqeust
